Updates README.md with error reporting code snippet (#53)
PSPDFKit/Cordova-Android: README.md

## Troubleshooting

### Error Reporting
To get a proper error reporting you need to register a global error listener which will print most of runtime errors into Logcat.

> Put the following code snippet into `[your-project]/platforms⁩/android⁩/app⁩/src⁩/main⁩/assets⁩/⁨www⁩/js⁩/index.js`:

Reviewer: This is on a new line. Do you want to have paragraph here? Can you add empty line before?

## Troubleshooting

### Error Reporting

> To get a proper error reporting you need to register a global error listener which will print most of runtime errors into Logcat.

Reviewer 2: Rewording: 

```suggestion
To get proper error reporting for JavaScript exceptions, you need to register a global error listener which will print most runtime errors to Logcat.
```
